src/Slider.js

```diff
@@ -24,11 +20,6 @@ class Slider extends React.Component {
             <input
                 className={classes}
                 type="range"
-                min={min}
-                max={max}
-                value={value}
-                tabIndex="0"
```

Comment:
Yep, I think it's the default value. Might be better to ask upstream why they specify the tabIndex value then before changing it here. Because they have it in all their demos.